roles/setup_pgd/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
I don't think a README should be added simply as place holder. :)

Reply:
It's WIP. I haven't finished testing yet. I will add README.